frontend/src/store.ts

```diff
@@ -0,0 +1,5 @@
+import { writable } from 'svelte/store';
+
+export const nonInteractiveState = writable<boolean>(true);
```

Comment:
why does this exist? cant you just pass values down from App.svelte?

Reply:
I need this variable to be used between App.svelte and G6Graph, to switch the nonInteractiveState on and off

Reply:
but cant G6graph take that as an exported value? so that App.svelte can pass it down?

Reply:
once we pass down the variable we can't access it or change it's value again in G6Graph and we need to be able to do that when disabling the mode on graph interactions. I needed a way where both App and G6Graph could both be able to access this variable.

Reply:
You need to bind the value then
see: https://svelte.dev/tutorial/component-bindings

Reply:
can you elaborate?

Reply:
> once we pass down the variable we can't access it or change it's value again

You need to bind the value while doing this, then it will work